setup.cfg

```diff
@@ -34,6 +34,7 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
+ignore-names = X, Y, X_train, X_test, Y_col
```

Comment:
Most of this names are very standard and acceptable, but I think this is not true for `Y_col`.
If it is not a huge change, I'd prefer to remove this from the list and adapt the corresponding code.